src/lib/core/datetime/date-adapter.ts

```diff
@@ -192,6 +187,26 @@ export abstract class DateAdapter<D> {
   abstract isValid(date: D): boolean;
 
   /**
+   * Attempts to deserialize a value to a valid date object. This is different from parsing in that
+   * deserialize should only accept non-ambiguous, locale-independent values (e.g. a ISO 8601
```

Comment:
"locale-independent values" -> "locale-independent formats"?

Does the phrase "locale-independent" refer specifically to the format? E.g., ISO 8601 not a localized format, but it might not contain UTC